### PR TITLE
Gallery review from Marin

### PR DESCRIPTION
**Pull request by icanzilb**

### CODE REVIEW

File: 08-beginner-actor/projects/final/EmojiArt/Model/Utility.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@ extension Task where Success == Never, Failure == Never {
   /// Suspends the current task for at least the given duration in seconds.
   /// - Parameter seconds: The sleep duration in seconds.
   static func sleep(seconds: TimeInterval) async {
-    await Task.sleep(UInt64(seconds * 1_000_000_000))
+		try? await Task.sleep(nanoseconds: UInt64(seconds * 1_000_000_000))
```
Comment (freak4pc) [inline 08-beginner-actor/projects/final/EmojiArt/Model/Utility.swift]: indentation

File: 08-beginner-actor/projects/starter/EmojiArt/Model/Utility.swift [hunk 2]
```diff
@@ -44,7 +44,7 @@ extension Task where Success == Never, Failure == Never {
   /// Suspends the current task for at least the given duration in seconds.
   /// - Parameter seconds: The sleep duration in seconds.
   static func sleep(seconds: TimeInterval) async {
-    await Task.sleep(UInt64(seconds * 1_000_000_000))
+		try? await Task.sleep(nanoseconds: UInt64(seconds * 1_000_000_000))
```
Comment (freak4pc) [inline 08-beginner-actor/projects/starter/EmojiArt/Model/Utility.swift]: indentation

File: 09-intermediate-actor/projects/challenge/EmojiArt/Model/Utility.swift [hunk 3]
```diff
@@ -44,7 +44,7 @@ extension Task where Success == Never, Failure == Never {
   /// Suspends the current task for at least the given duration in seconds.
   /// - Parameter seconds: The sleep duration in seconds.
   static func sleep(seconds: TimeInterval) async {
-    await Task.sleep(UInt64(seconds * 1_000_000_000))
+		try? await Task.sleep(nanoseconds: UInt64(seconds * 1_000_000_000))
```
Comment (freak4pc) [inline 09-intermediate-actor/projects/challenge/EmojiArt/Model/Utility.swift]: indentation

File: 09-intermediate-actor/projects/final/EmojiArt/Model/Utility.swift [hunk 4]
```diff
@@ -44,7 +44,7 @@ extension Task where Success == Never, Failure == Never {
   /// Suspends the current task for at least the given duration in seconds.
   /// - Parameter seconds: The sleep duration in seconds.
   static func sleep(seconds: TimeInterval) async {
-    await Task.sleep(UInt64(seconds * 1_000_000_000))
+		try? await Task.sleep(nanoseconds: UInt64(seconds * 1_000_000_000))
```
Comment (freak4pc) [inline 09-intermediate-actor/projects/final/EmojiArt/Model/Utility.swift]: indentation

File: 09-intermediate-actor/projects/starter/EmojiArt/Model/Utility.swift [hunk 5]
```diff
@@ -44,7 +44,7 @@ extension Task where Success == Never, Failure == Never {
   /// Suspends the current task for at least the given duration in seconds.
   /// - Parameter seconds: The sleep duration in seconds.
   static func sleep(seconds: TimeInterval) async {
-    await Task.sleep(UInt64(seconds * 1_000_000_000))
+		try? await Task.sleep(nanoseconds: UInt64(seconds * 1_000_000_000))
```
Comment (freak4pc) [inline 09-intermediate-actor/projects/starter/EmojiArt/Model/Utility.swift]: indentation

**Comment by icanzilb** [comment]
@freak4pc I hate the 2 spaces rule SO MUCH ... fixed

**Review by freak4pc — APPROVED** [review]

**Pull request merged**